Index: src/navigation/screens/app/index.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>export * from \"./HomeScreen\"
===================================================================
diff --git a/src/navigation/screens/app/index.ts b/src/navigation/screens/app/index.ts
--- a/src/navigation/screens/app/index.ts	(revision c621a811bb882733fc4c1390185012bd0cb8dbc6)
+++ b/src/navigation/screens/app/index.ts	(date 1741980170962)
@@ -1,1 +1,2 @@
-export * from "./HomeScreen"
\ No newline at end of file
+export * from "./HomeScreen"
+export * from "./ScanScreen"
\ No newline at end of file
